.github/workflows/deploy_tre.yml

```diff
@@ -133,6 +133,7 @@ jobs:
             export TF_VAR_resource_processor_client_id=$RESOURCE_PROCESSOR_CLIENT_ID
             export TF_VAR_resource_processor_client_secret=$RESOURCE_PROCESSOR_CLIENT_SECRET
             export TF_VAR_debug="false"
+            export TF_VAR_deploy_gitea=true
```

Comment:
This means that we always want to deploy that service, right? Shouldn't this be an option a user need to choose?

Reply:
This is only in our CD pipeline. The person deploying can set it to false if they do not want deploy.

Reply:
Also if not clear, this is a shared service, so decision at deploy time, rather than a service within a workspace.

Reply:
@marrobi, the "DEPLOY_GITEA" var should also be documented in the docs under both manual deployment and workflows